packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -11,6 +11,33 @@ const outputDir = Path.resolve(__dirname, '../dist/');
 
 const WIDGET_VERSION = findLatestWidgetVersion(signInWidgetMajorVersion);
 
+function configUris() {
+  let uris;
+  switch (process.env.PROD_ENV) {
```

Comment:
`switch` does not make sense here -- We should use `if(process.env.PROD_ENV)` since the variable is boolean

Reply:
in the last update, ENV has been modified now it could be equal to `prod` or `test` (in further maybe it could be extended) so is it worth to keep switch case construction?

Reply:
Cool, that makes sense with the new changes :)